staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -690,7 +684,16 @@ func (r *crdHandler) getOrCreateServingInfoFor(uid types.UID, name string) (*crd
 		}
 	}
 
-	safeConverter, unsafeConverter, err := r.converterFactory.NewConverter(crd)
+	var converter conversion.CRConverter
+	if r.converterFactory == nil {
+		converter = conversion.NewNOPConverter()
```

Comment:
Is this used/useful? Would it be better to return an error if the provided `converterFactory` is nil?

Reply:
Hmm, thinking about this more, this method is part of serving a request, so you'd get the error when you interact with CRDs/CRs. I wonder if it would make more sense to check for this earlier, in `NewCustomResourceDefinitionHandler`?

Reply:
+1 to returning an error in NewCustomResourceDefinitionHandler if converterFactory is nil, and assuming this is non-nil here, rather than turning wiring errors into maybe-data-destroying no-op conversions or request-time errors